src/components/TvDetails/index.tsx

```diff
@@ -200,6 +201,7 @@ const TvDetails = ({ tv }: TvDetailsProps) => {
     );
   }
 
+  // Does NOT include "Specials"
   const seasonCount = data.seasons.filter(
     (season) => season.seasonNumber !== 0 && season.episodeCount !== 0
```

Comment:
I might be wrong but can't we simplify this to:
```suggestion
    (season) => season.episodeCount !== 0
```
Then we have no need for `showHasSpecials`?

Reply:
Good spot @westdom . Yes, you are right, and that's what I did initially, but I made the intentional decision of ensuring that the `seasonCount` variable does not include TV Show Specials.

Why? Because in the header / banner of every TV Show, it is written how many seasons are in the TV Show in question. I made the intentional decision that, for example, if a TV Show like `Doctor Who` has 13 Seasons + Specials, then we wouldn't say `Doctor Who` has 14 Seasons, as for the average user, they don't really consider specials as a separate season. Hence, it was important that `seasonCount` did not include TV Show Specials in its count.

Because of this, the `showHasSpecials` variable was needed.

Reply:
Perfect that makes sense now, thanks!